catalog/utilities/media_props_gen/docs/media_props.md

```diff
@@ -2,424 +2,376 @@
+## Object Shape
 
-## Existing Data Inconsistencies
+This is a keyword for the provider, a string in a "snake_case" form.
 
 # source
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
+The name of the source of the media item.It can be a collection on a provider
+site, or the provider itself.
 
-## Normalization and Validation
+## Object Shape
 
-## Existing Data Inconsistencies
+This is a keyword for the source, a string in a "snake_case" form.
 
 # foreign_identifier
 
 ## Description
 
-## Object Shape
+The unique identifier for the media item on the source site.
 
 ## Selection Criteria
 
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+Ideally, this identifier should be usable to generate the URL to the media item
+on the source site.
 
 # foreign_landing_url
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
-
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+The URL of the landing page for the media item (not a direct link to the media
+file). This should be unique for each media item.
 
 # url
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
-
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+The direct URL to the media file, from which the media file can be downloaded.
+This should be unique for each media item.
 
 # thumbnail
 
 ## Description
 
-## Object Shape
+The URL of a smaller, thumbnail, image for the media item.
 
 ## Selection Criteria
 
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+The smallest acceptable size for a thumbnail is 600px at the longest edge (see
+https://github.com/WordPress/openverse/issues/675#issuecomment-1472399310)
 
 # width
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
-
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+The width of the main image in pixels.
 
 # height
 
 ## Description
 
-## Object Shape
+The height of the main image in pixels.
 
 ## Selection Criteria
 
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+If the provider does not provide the height and width of the image, it is
+possible to send a head request to the direct url to extract this dat.
 
 # filesize
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
-
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+The size of the main media file in bytes. If not available in the API response,
+it can be extracted from a head request response to the media file URL.
 
 # license
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
-
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+The slug of the license under which the media item is licensed. For the list of
+available license slugs, see
+[openverse-attribution package](https://github.com/WordPress/openverse/blob/3329d307f5cdce0cbe1f1c13dbe17106970abdaa/packages/python/openverse-attribution/src/openverse_attribution/license_name.py)
```

Comment:
One nit, and it might also be best to reference this by `main` since this document may not get updated all that often.

```suggestion
[openverse-attribution package](https://github.com/WordPress/openverse/tree/main/packages/python/openverse-attribution/src/openverse_attribution/license_name.py).
```